pdr_backend/cli/cli_module.py

```diff
@@ -163,3 +164,22 @@ def do_publisher(args):
 def do_topup(args):
     ppss = PPSS(yaml_filename=args.PPSS_FILE, network=args.NETWORK)
     topup_main(ppss)
+
+
+@enforce_types
+def do_create_accounts(args):
+    create_accounts(args.NUM)
+
+
+@enforce_types
+def do_view_accounts(args):
+    ppss = PPSS(yaml_filename=args.PPSS_FILE, network=args.NETWORK)
+    accounts = args.ACCOUNTS.split(",")
```

Comment:
hmm... I think by this point they are already converted to a list, right?

Reply:
yes, i've updated all of this and a lot more

check other cli_arguments implementations and modules that were using 0x addresses, and are all passed in via an `address_str` structure